### PR TITLE
Hammerhead

### PR DESCRIPTION
**Pull request by francinum**
Hammerhead Minimap

## About The Pull Request

Adds a ship meant to be feature-complete, sacrificing as little as possible, besides multiple Z-Levels.

## Progress

- [X] Greybox All Departments
- [x] Power
- [x] Atmos
- [x] Full Ship Systems

## Changelog
:cl:
add: Hammerhead Capital Ship
balance: The Hammerhead is a slow, well-armored behemoth. It's also actually nice to navigate without a webmap.
/:cl:

### CODE REVIEW

**Comment by francinum** [comment]
ah hell there's git garbage all over my PR.

**Comment by Karmic-Skink** [comment]
What did you do?

**Comment by francinum** [comment]
I blame fighters

**Comment by Jalleo** [comment]
I have given a bunch of suggestions to this map to help make it 
**_perfectly balanced_**
======

**Comment by GoatMonarch** [comment]
Allright, so while not perfect this map's an improvement over the Aegis by the fact it actually has departments already planned out and implemented. Such as xenobio (thank god), and atmos being fleshed out.

**Comment by francinum** [comment]
Botany is being fully remapped and replacing the strange bridge maintenance closet entirely.
The grilles have been removed.
Maintenance will be properly fluffed and equipped during the final pass.

**Comment by francinum** [comment]
Awaiting whatever KMC has planned for his edits.

File: nsv13/code/game/area/xerxes.dm [hunk 1]
```diff
@@ -0,0 +1,46 @@
+/area/ai_monitored/security/armory/lockup
```
Comment (Kmc2000) [inline nsv13/code/game/area/xerxes.dm]: Xerxes.dm -> hammerhead.dm

**Pull request merged**